packages/notebook/src/model.ts

```diff
@@ -231,6 +231,10 @@ export class NotebookModel implements INotebookModel {
     return info?.name ?? '';
   }
 
+  get collaborative(): boolean {
```

Comment:
```suggestion  
  /**
   * Whether the model is collaborative or not.
   */
  get collaborative(): boolean {
```

Same about the value here - it should depend on the value of `collaborationEnabled` pass to the constructor.